tests/plugins/colordialog/manual/addhash.md

```diff
@@ -0,0 +1,13 @@
+@bender-tags: tc, feature, colordialog, 565, 4.8.0
```

Comment:
It was discussed around 12 - 13 June on `ckeditor4-chat`. I made some kind of sum up for us as pinned post to ckeditor4-chat.